src/app/theme-switcher/theme-switcher.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-theme-switcher',
+  templateUrl: './theme-switcher.component.html',
+  styleUrls: ['./theme-switcher.component.scss']
+})
+export class ThemeSwitcherComponent {
+
+  isLightTheme: boolean = false;
+
+  onThemeSwitchChange(): void {
+    this.isLightTheme = !this.isLightTheme;
+
+    document.body.setAttribute(
```

Comment:
constructor(@Inject(DOCUMENT) private document: Document, private renderer2: Renderer2){}
.....
this.renderer2.setAttribute(this.document.body, 'data-bs-theme', this.isLightTheme ? 'light' : 'dark')

Reply:
Why is it important?